addons/website/static/src/snippets/s_facebook_page/000.js

```diff
@@ -24,10 +22,39 @@ const FacebookPageWidget = publicWidget.Widget.extend({
             params.href = `https://www.facebook.com/${params.id}`;
         }
         delete params.id;
+
+        this._renderIframe(params);
+
+        this.resizeObserver = new ResizeObserver(debounce(this._renderIframe.bind(this, params), 100));
+        this.resizeObserver.observe(document.querySelector("#wrapwrap"));
+
+        return def;
+    },
+    /**
+     * @override
+     */
+    destroy: function () {
+        this._super.apply(this, arguments);
+
+        this.options.wysiwyg && this.options.wysiwyg.odooEditor.observerUnactive();
+        if (this.iframeEl) {
+            this.iframeEl.remove();
+        }
+        this.options.wysiwyg && this.options.wysiwyg.odooEditor.observerActive();
+        this.resizeObserver.disconnect();
+    },
+    /**
+     * _renderIframe: prepare iframe element & replace it with existing iframe
```

Comment:
no need to indicate the function name here..
```suggestion
     * Prepares iframe element & replace it with existing iframe.
```